File: Front-end/src/app/locations/airport.ts
import { BaseEntity } from '../shared/baseEntity';
import { Lookup } from '../lookups/lookup';
import { Country } from '../lookups/country';
import { City } from '../lookups/city';

export interface Airport extends BaseEntity {
		/** public int AirportId { get; set; } */
		name: string;
		type: Lookup;
		place: {
			country: Country;
			state: {
				stateId: number;
				name: string;
			};
			city: City;
			other: string;
		};
		iataCode: string;
		code: string;
		location: Location;
		elevation?: number;
}

File: Service/Z2data.Core/Domain/Locations/Airport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Z2data.Core.Domain.Events;
using Z2data.Core.Domain.Lookups;

namespace Z2data.Core.Domain.Locations
{
    public class Airport : BaseEntity
    {
        //public int AirportId { get; set; }
        public string Name { get; set; }
        public Lookup Type { get; set; }
        public Place Place { get; set; }
        public string IataCode { get; set; }
        public string Code { get; set; }
        public Location Location { get; set; }
        public double? Elevation { get; set; }
    }
}
